Update for jsonb compression (#4566)
timescale/docs: use-timescale/hypercore/compression-methods.md

original data. TimescaleDB automatically detects this case, and falls back to
not using a dictionary in that scenario.

#### JSONB compression

> For JSONB columns, TimescaleDB uses a two-layer compression approach:

Reviewer: ```suggestion
For JSONB columns, $TIMESCALE_DB uses a two-layer compression approach:
```

> 1. **TimescaleDB dictionary compression**: First, TimescaleDB attempts to apply

Reviewer: ```suggestion
1. **$TIMESCALE_DB dictionary compression**: $TIMESCALE_DB attempts to apply
```

   its own dictionary compression to the JSONB data. This works well when JSONB

>    values have high repetition.

Reviewer: ```suggestion
   values have high repetition. It does not perform when each JSONB value is unique or nearly unique.
```

> 2. **PostgreSQL TOAST compression**: If dictionary compression doesn't perform

Reviewer: ```suggestion
2. **$PS TOAST compression**: If dictionary compression does not  perform, $TiimescaleDB skips to enable 
```

>    well (for example, when each JSONB value is unique or nearly unique),

Reviewer: remove

>    TimescaleDB skips compression and allows PostgreSQL to handle the data using

Reviewer: ```suggestion
  $PG to handle the data using
```

>    its TOAST (The Oversized-Attribute Storage Technique) compression. By default,

Reviewer: ```suggestion
   Oversized-Attribute Storage Technique (TOAST) compression. By default,
```

>    PostgreSQL uses `pglz` compression, but this can be configured to use more

Reviewer: ```suggestion
   PostgreSQL uses `pglz` compression. To configure  $PG to use more 
```

>    modern compression algorithms like `lz4` by setting the `default_toast_compression`

Reviewer: ```suggestion
   modern compression algorithms like `lz4`, set the `default_toast_compression`
```

   configuration parameter.

> You can check your current TOAST compression setting:

Reviewer: ```suggestion
To check your current TOAST compression setting:
```

```sql
SHOW default_toast_compression;
```

This two-layer approach ensures that JSONB data is compressed efficiently

> regardless of its characteristics, using dictionary compression when beneficial

Reviewer: ```suggestion
regardless of its characteristics.
```

> and falling back to LZ-based compression when dictionary compression is not effective.

Reviewer: remove
